+srv/expServer.m

```diff
@@ -359,13 +359,14 @@ function whiteScreen()
   function calibrateGamma()
     stimWindow = rig.stimWindow;
     DaqDev = rig.daqController.DaqIds;
-    lightIn = 'ai0'; % defaults from hw.psy.Window
-    clockIn = 'ai1';
-    clockOut = 'port1/line0 (PFI4)';
-    log(['Please connect photodiode to %s, clockIn to %s and clockOut to %s.\r'...
-        'Press any key to contiue\n'],lightIn,clockIn,clockOut);
+    lightIn = 'ai1'; % defaults from hw.psy.Window
```

Comment:
@nsteinme why did you switch the `lightin` and `clockIn` channels here? I just ran gamma calibration with the old channels and it worked fine (though it also works with your suggested channels)

Reply:
following up on this

Reply:
As this doesn't affect things I'm happy to go ahead with this merge.  I will fix the typo on line 362 first though...